src/ingest-pipeline/airflow/dags/launch_multi_analysis.py

```diff
@@ -17,12 +17,14 @@
     get_soft_data_type
 )
 
+from extra_utils import check_link_published_drvs
+
 
 def get_uuid_for_error(**kwargs):
     """
     Return the uuid for the derived dataset if it exists, and of the parent dataset otherwise.
     """
-    return None
+    return ''
```

Comment:
Can we get rid of this function and use a lambda instead in 'on_failure_callback'?